resources/views/database/links.blade.php

```diff
@@ -6,34 +6,62 @@
             <a href="{{route("database")}}" class="btn btn-primary col-md-2"> <i class="fas fa-arrow-circle-left"></i> Dashboard </a>
         </div>
         <h2 class="text-center"> Links </h2>
-        <table class="table w-50 mx-auto mt-4">
-            <thead>
-            <tr>
-                <th scope="col">#</th>
-                <th scope="col">Link</th>
-                <th scope="col" colspan="2">Actions</th>
-            </tr>
-            </thead>
-            <tbody>
-            @foreach($links as $link)
+        <div class="table-responsive text-align-center">
+            <table class="table w-auto mx-auto mt-4">
+                <thead>
                 <tr>
-                    <th scope="row">{{$loop->index+1}}</th>
-                    <td>{{$link->linkText}}</td>
-                    <td>
-                        <form method="POST" action="{{ route('deleteLink') }}">
-                            {{ csrf_field() }}
-                            <input id="delete" name="delete" value="delete" hidden>
-                            <input id="linkId-{{$link->link_id}}" name="link_id" value="{{$link->link_id}}" hidden>
-                            <button type="submit" class="btn btn-danger">Delete</button>
-                        </form>
-                    </td>
-                    <td>
-                        <a href="{{route('modifyLink', ['link_id' => $link->link_id])}}" class="btn btn-primary"><i class="fas fa-edit" aria-hidden="true"></i> Modify </a>
-                    </td>
+                    <th scope="col">#</th>
+                    <th scope="col">Name</th>
+                    <th scope="col">Link Text</th>
+                    <th scope="col">Status</th>
+                    <th scope="col">Last Update</th>
+                    <th scope="col" colspan="2">Actions</th>
                 </tr>
-            @endforeach
-            </tbody>
-        </table>
+                </thead>
+                <tbody>
+                @foreach($links as $link)
+                    <tr>
+                        <th scope="row">{{$loop->index+1}}</th>
+                        <td>{{$link->name}}</td>
+                        <td>{{$link->linkText}}</td>
+                        <td>{{$link->status}}</td>
```

Comment:
For the link status, let's put it into a Bootstrap pill: 
`<span class="badge badge-success">{{$link->status}}</span>`
and then adjust the color based on the status (`badge-success` for Valid, `badge-warning` for Unknown, and `badge-danger` for Broken.